python/pfs/drp/stella/fitPfsFluxReference.py

```diff
@@ -574,6 +583,187 @@ def fitModelsToSpectra(
         """For each observed spectrum,
         get probability of each model fitting to the spectrum.
 
+        Parameters
+        ----------
+        pfsConfig : `pfs.datamodel.pfsConfig.PfsConfig`
+            Configuration of the PFS top-end.
+        obsSpectra : `PfsFiberArraySet`
+            Continuum-subtracted observed spectra
+        pfsMergedLsf : `dict` (`int`: `pfs.drp.stella.Lsf`)
+            Combined line-spread functions indexed by fiberId.
+        radialVelocities : `list` of `Optional[lsst.pipe.base.Struct]`
+            Radial velocity for each fiber.
+            Each element, if not None, has ``velocity``, ``error``, and ``fail``
+            as its member. See ``EstimateRadialVelocityTask``.
+        priorPdfs : `list` of `numpy.array` of `float`
+            For each ``priorPdfs[iSpectrum]`` in ``priorPdfs``,
+            ``priorPdfs[iSpectrum][iSED]`` is the prior probability of the SED ``iSED``
+            matching the spectrum ``pfsConfig.fiberId[iSpectrum]``.
+            ``priorPdfs[iSpectrum]`` can be ``None``,
+            in which case the corresponding return value will be ``None``.
+
+        Returns
+        -------
+        bestParams : `list` of `Struct|None`
+            ``bestParams[iSpectrum]`` is the best-fit parameters for
+            ``pfsConfig.fiberId[iSpectrum]``. Each element ``bestParams[iSpectrum]``,
+            if not None, consists of the following members:
+
+              - teff : `float`
+                "Teff" parameter.
+              - logg : `float`
+                "log(g)" parameter.
+              - m : `float`
+                metalicity parameter.
+              - alpha : `float`
+                "alpha" parameter.
+              - success : `bool`
+                True if optimization has succeeded.
+
+            The following members exist if self.config.minimizationMethod != 'brute-force'
+
+              - chi2 : `float`
+                chi square.
+              - dof : `int`
+                degree of freedom of the chi square.
+        """
+        if self.config.minimizationMethod.lower() == "brute-force":
+            return self.fitModelsToSpectraBruteForce(
+                pfsConfig, obsSpectra, pfsMergedLsf, radialVelocities, priorPdfs
+            )
+
+        # Value by which to multiply T_eff to adjust it to be as large as the other parameters.
+        teffScale = 1e-3
```

Comment:
Make this a config parameter?

Reply:
I decide to use FluxModelInterpolator's `teffScale`, `loggScale`, ... here instead of using the immediate value.